ui/ui-frontend/projects/archive-search/src/app/archive/management-rules/archive-unit-rules/update-unit-rules/update-unit-rules.component.ts

```diff
@@ -412,13 +409,15 @@ export class UpdateUnitRulesComponent implements OnInit, OnDestroy {
       this.searchArchiveUnitsByCriteriaSubscription = this.archiveService
         .searchArchiveUnitsByCriteria(this.criteriaSearchDSLQuery, this.accessContract)
         .subscribe((data) => {
-          if (data.totalResults === RESULTS_MAX_NUMBER) {
+          if (data.totalResults === ArchiveSearchConstsEnum.RESULTS_MAX_NUMBER) {
             this.itemsWithSameRule = data.totalResults.toString();
             this.itemsToUpdate = this.resultNumberToShow;
           } else {
             this.itemsWithSameRule = data.totalResults.toString();
```

Comment:
a chaque fois this.itemsWithSameRule = data.totalResults.toString(); est dans le if et le else, c'est nortmal ?

Reply:
duplication supprimée, et bloc de traitement modifié